### ISSUE TITLE
Auto Unselect

### ISSUE THREAD

**Issue by tikaPahadi** [hyperoslo/ImagePicker#266]
Example with Configuration of Maximum 1 Pickup:
When an image is selected, I think it should un-select if any exist. It can be frustrating when you select one picture from far scrolled and you want to select another. You have to **unselect** and select another.

**Comment by acjh** [comment]
Duplicate of #174 and #200 
Feature added in PR #245

**Comment by tikaPahadi** [comment]
I don not believe this property is working. First, `Configuration.allowMultiplePhotoSelection = false` does not exist. Then, I do not see when this feature was added.

**Comment by zenangst** [comment]
@tikaPahadi ohai, could you clarify what you mean that it doesn't exist? Mind sharing which version of `ImagePicker` you are currently using? You could point to the `master` branch to get the latest features and fixes just check it out. We will make a new release shortly 😎 Let us know how it goes!

**Comment by tikaPahadi** [comment]
@zenangst I looked at the source code and realized that Configuration is no longer static property. I was using Pods release which has no allowMultiplePhotoSelection. But the latest git pull works for me. Although the information in the Readme is **misleading** about this, I have created [a small commit](https://github.com/tikaPahadi/ImagePicker/commit/97b3c1efe20f47d61f8ab2f336a1b8fe53cad4aa) to update the document if you are interested in it.
One again, kudos to the team!
